whelk-core/src/main/groovy/whelk/component/ElasticSearch.groovy

```diff
@@ -501,7 +468,56 @@ class ElasticSearch {
         })
     }
 
-    private static void setComputedProperties(Document doc, Set<String> links, Whelk whelk) {
+    private static void setComputedProperties(Document doc, Map framed, Set<String> links, Whelk whelk) {
```

Comment:
Now that this method takes both framed and doc (copy) it is not on different abstraction level than the caller. I think it should be inlined in getShapeForIndex